File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/Sys_Configurations.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_Configurations } from './Sys_Configurations.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_ConfigurationsService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'AllowNullUserPassword', 'ChangePasswordOnFirstLogon', 'IsUsingStrongPassword', 'UserPasswordExpiryDay', 'TimesOfHistoryPasswordNotAllow', 'MinPasswordLen', 'PasswordExpiredNotifyDay'];

    List: Sys_Configurations[] | undefined;
    ListFilter: Sys_Configurations[] | undefined;
    FormData!: Sys_Configurations;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_Configurations";
    }
}

File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/Sys_ConfigurationsService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class Sys_ConfigurationsService : BaseService<Sys_Configurations, ISys_ConfigurationsRepository>
    , ISys_ConfigurationsService
    {
    private readonly ISys_ConfigurationsRepository _Sys_ConfigurationsRepository;
    public Sys_ConfigurationsService(ISys_ConfigurationsRepository Sys_ConfigurationsRepository) : base(Sys_ConfigurationsRepository)
    {
    _Sys_ConfigurationsRepository = Sys_ConfigurationsRepository;
    }
    }
    }

